AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/AuthDirective/GraphQLAuthDirectiveIntegrationTests+Support.swift

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+import AWSMobileClient
+import AWSAPICategoryPlugin
+@testable import AWSAPICategoryPluginTestCommon
+@testable import AmplifyTestCommon
+import AWSPluginsCore
+
+extension GraphQLAuthDirectiveIntegrationTests {
+    func createNote(_ id: String = UUID().uuidString,
+                    content: String) -> Result<MutationSyncResult, GraphQLResponseError<MutationSyncResult>> {
+        let createdNoteInvoked = expectation(description: "note was created")
+        var result: Result<MutationSyncResult, GraphQLResponseError<MutationSyncResult>>?
+        let note = SocialNote(id: id, content: content, owner: nil)
+        let request = GraphQLRequest<MutationSyncResult>.createMutation(of: note)
+        _ = Amplify.API.mutate(request: request, listener: { event in
+            switch event {
+            case .completed(let data):
+                switch data {
+                case .success(let note):
+                    result = .success(note)
+                case .failure(let error):
+                    result = .failure(error)
+                }
+                createdNoteInvoked.fulfill()
+            case .failed(let error):
+                XCTFail("Got failed, error: \(error)")
+            default:
+                XCTFail("Unexpected event: \(event)")
+            }
+        })
+
+        wait(for: [createdNoteInvoked], timeout: TestCommonConstants.networkTimeout)
+        return result!
+    }
+
+    func updateNote(_ note: SocialNote,
+                    version: Int) -> Result<MutationSyncResult, GraphQLResponseError<MutationSyncResult>> {
+        let updateNoteInvoked = expectation(description: "note was update")
```

Comment:
s/update/updated/ ?

Reply:
updated, only the description, i think the variable ends with "Invoked" is already past tense